scripts/call_monitor_submission_demo.py

```diff
@@ -0,0 +1,34 @@
+"""Creating and Running Monitor Submission Workflow."""
+from firecloud import api as fapi
+from firecloud import errors as ferrors
+
+workspace_name = 'HCA_Optimus_Pipeline_MS'
+workspace_project = 'broad-firecloud-dsde'
+submission_id = '990e5930-a9b8-4f70-b220-11b7ffe702bb'
+# workflow_name has to be unique everytime
+workflow_name = 'monitor_submission_22'
+
+json_body = {
```

Comment:
where did this json come from? can it be retrieved using an API rather than hard-coded in the script? e.g. pretty sure we don't want `"namespace": 'horsefish'` hard coded here

Reply:
I made the JSON, with the comments from fiss.apy.py script, input.json, and playing around

Reply:
hmm, then can you populate its contents more programmatically? e.g. 'horsefish' shouldn't be hard-coded anywhere, nor should methodConfigVersion be hardcoded as 14, etc etc